fileio/simpleio.go

```diff
@@ -115,7 +115,12 @@ func CatchErrThrowEOF(err error) {
 
 // BytesToBuffer will parse []byte and return a pointer to the same underlying bytes.Buffer
 func BytesToBuffer(reader *ByteReader) *bytes.Buffer {
-	_, err := reader.Buffer.Write(reader.line[:len(reader.line)-1])
+	var err error
```

Comment:
I see that there is no filelio/simpleio.go in gonomics, and this is producing the conflict with the merge. It looks closest to byteio.go
Would the solution be for Olivier to go back, rename the file to correct name, and then do another pull request? I'm confused by git didn't just try to make this into a new file since it's missing versus calling a conflict?